Create docathon page for docathon 2021 (#786)
mattermost/mattermost-handbook: operations/research-and-development/product/technical-writing/docathon-2021.md

# Documentation Hackathon 2021

The first Mattermost Docathon is being held from July 26th to August 6th 2021. Take a look at our [blog post](https://mattermost.com/blog/docathon-2021/) if you haven't already.

## What is a Docathon?

A docathon is a hackathon for product documentation. 

## Why a Docathon? 

> Documentation isn't written in a vacuum and it's important to see the customer’s documentation journey through the eyes of our Mattermost community. The best way to do that is to get our community involved! Documentation is also often a great way to get started on your open source contribution journey as it's usually how you get to know the product.

Reviewer: ```suggestion
Documentation isn't written in a vacuum, and it's important to see the customer’s documentation journey through the eyes of our Mattermost community. The best way to do that is to get our community involved! Documentation is also a great way to get started on your open source contribution journey as it's usually how you get to know the product.
```

## Who can participate?

The Docathon is designed to provide opportunities for all contributors to get involved, whether new to open source or seasoned contributors. Help us solve technical tooling challenges. Write mobile steps for Mattermost messaging tasks. Correct inaccuracies and inconsistencies. Add or update screenshots, labeled images, animated GIFs, and videos. How do you want to improve the Mattermost product documentation? 

## Can Mattermost staff help?

Absolutely! Here are some ways you can get involved:

> * **Help us merge PR submissions:** This will basically just entail helping us go through the PRs with the Documentation 2021 label, making sure they follow our style guide and fix any grammatical issues. This isn't an extremely rigorous process - we just want to make sure anything merged follows some standardization.

Reviewer: ```suggestion
* **Help us merge PR submissions:** This includes helping us go through the PRs with the **Docathon 2021** label, making sure PRs follow our style guide, and fixing any grammatical issues. This isn't an extremely rigorous process - we just want to make sure anything merged follows some standardization.
```

> * **Help review any technical content for accuracy:** Be actively monitoring the ~Docathon 2021 channel for any questions, and just basically being engaging to anyone who posts. Also being open for participants to DM you if they have questions they dont want to ask in the public channel. Also help relay any feedback to us.

Reviewer: ```suggestion
* **Help review technical content for accuracy:** Actively monitor the [~Docathon 2021](https://community-daily.mattermost.com/core/channels/docathon2021) channel, respond to questions, and engage community members who post with questions or concerns. You can also help by being open to contributors messaging you directly if they have questions and prefer not to ask in the public channel, and relay any feedback back to the Technical Writing team.
```

> * **Help us endorse and/or write recommendations for people on LinkedIn:** This is especially welcome if you help with reviewing and there's someone you personally feel compelled to network with and help out their professional career.

Reviewer: ```suggestion
* **Help us with Linked In recommendations or endorsements for top contributors:** This is particularly welcome in cases where, during PR reviews, you encounter Docathon contributors you want to network with and whose professional career you want to support.
```

> * **Contribute your own content or ideas for either the docs overall, or the event itself!**

Reviewer: ```suggestion
* **Contribute your own content or ideas for either Mattermost product documentation, or the Docathon event itself!**
```

## Who can participate?

The Docathon is designed to provide opportunities for all contributors to get involved, whether new to open source or seasoned contributors. Help us solve technical tooling challenges. Write mobile steps for Mattermost messaging tasks. Correct inaccuracies and inconsistencies. Add or update screenshots, labeled images, animated GIFs, and videos. How do you want to improve the Mattermost product documentation? 

## Can Mattermost staff help?

Absolutely! Here are some ways you can get involved:

* **Help us merge PR submissions:** This will basically just entail helping us go through the PRs with the Documentation 2021 label, making sure they follow our style guide and fix any grammatical issues. This isn't an extremely rigorous process or anything, we just want to make sure anything merged follows some standardization.
* **Help review any technical content for accuracy:** Be actively monitoring the ~Docathon 2021 channel for any questions, and just basically being engaging to anyone who posts. Also being open for participants to DM you if they have questions they dont want to ask in the public channel. Also help relay any feedback to us

> ^^  This is where i think the most value is overall, because I want everyone who participates to feel as if their efforts are being seen and that this is a positive, fun event for them

Reviewer: Consider changing voice here to avoid the "I" focus.
